modules/nw-ovn-kubernetes-rollback.adoc

```diff
@@ -189,7 +171,26 @@ do
 done
 ----
 
-. After the nodes in your cluster have rebooted, start all of the machine configuration pools:
+
+. Wait until the Multus daemon set rollout completes.
++
+[source,terminal]
+----
+$ oc -n openshift-multus rollout status daemonset/multus
+----
++
+The name of the Multus pods is in form of `multus-<xxxxx>` where `<xxxxx>` is a random sequence of letters. It might take several moments for the pods to restart.
++
+.Example output
+[source,text]
+----
+Waiting for daemon set "multus" rollout to finish: 1 out of 6 new pods have been updated...
+...
+Waiting for daemon set "multus" rollout to finish: 5 of 6 updated pods are available...
+daemon set "multus" successfully rolled out
+----
+
+. After the nodes in your cluster have rebooted and the multus pods are rolled out, start all of the machine configuration pools:
```

Comment:
You may want to add something along the lines of the example below to keep parallelism throughout the procedure.
```suggestion
. After the nodes in your cluster have rebooted and the multus pods are rolled out, start all of the machine configuration pools by running the following commands:
```